### PR TITLE
🌿 [claude-inline-subtasks] codex: cover live tile lifecycle [step 7/9]

### PR DESCRIPTION
**Pull request by AlexDochioiu**
## Complexity
🌿 Straightforward: expanded lifecycle tests, removal of a test-only repository entry point, and documentation reconciliation.

## What
- Add preserved live write-path coverage for exact spawn fallback, child plaintext precedence, encrypted-input rejection, stable tile identity, busy accounting and lifecycle cleanup.
- Remove unused production `CodexMessageRepository.readMessages` and migrate all 25 test callers to the real prepare/project seam.
- Reconcile capability/regression docs and nine-step tracker with merged #1399 and recorded native QA limitations.

## Why
This completes the separated coverage slice for Codex live/replay tiles. It also resolves #1399's obsolete test-only read-path finding without adding duplicate child hydration logic.

## Risk and test focus
No user-visible or database impact. Production change only deletes a method with no production callers; live behavior and repository prepare/project methods remain unchanged. Test migration preserves root-only assumptions explicitly and retains all prior assertions. No runtime, native-adapter, client, shared-wire or scoped-stop changes.

## Expected result
Current tile lifecycle has broader deterministic coverage, and tests exercise supported repository seams. Scoped stop remains step 8/9; final matrix remains step 9/9, not completed by this PR.

## Verification
- Write-path suite: 54/54 passed.
- Rollout API suite: 58/58 passed; existing 274 expectations preserved.
- Full owning Codex suite, fatal-info analysis, formatting and whitespace checks passed.
- Independent correctness/simplicity review approved exact head `7b272f52e7`.
- Existing native-plugin QA was not rerun or relabeled: duplicate-name/plaintext/resumed-terminal live cases remain unexecuted, and direct app-server input to v2 children remains native-rejected.
- 599 changed lines (+378/-221), 11 files. Original coverage and full implementation snapshots preserved.

---
## Summary by cubic
Extends Codex live tile lifecycle coverage by preserving write-path regressions, removing the unused test-only `CodexMessageRepository.readMessages`, and reconciling docs with the merged integration PR #1399.

**Test coverage**
- Adds write-path cases for exact spawn fallback, child plaintext precedence, encrypted-input rejection, stable tile identity, busy accounting, and lifecycle cleanup.
- Migrates all 25 `readMessages` callers to the real prepare/project seam and deletes the method, which has no production callers.

No user-visible or database impact. Capability, regression, and tracker docs are updated to reflect merged #1399 and recorded native QA limitations, including naming the busy-accounting coverage at L2.

<sup>Written for commit d0e2c00644d7a0cc5ac15319d585693247b3a42f. Summary will update on new commits.</sup>

<a href="https://cubic.dev/pr/sesori-ai/sesori_apps_monorepo/pull/1420?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

File: docs/regression/notifications.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ external.
   with background sub-agents stays busy until the last one and its wake-up turn
   settle, so completion fires once for the whole span; a main-agent-only stop
   (`keep`) does not suppress the completion the kept sub-agents later earn,
-  while a full stop does.
+  while a full stop does. A Codex root likewise defers idle and completion while
```
[the remark this answers is not in the document]
Comment (AlexDochioiu) [reply docs/regression/notifications.md:22]: [Sesori reply] Added Codex live write-path busy-accounting coverage to L2 Routine: root remains busy while children run and emits one deferred idle after the last child settles. Documentation-only correction; whitespace check passed, unchanged passing tests were not rerun.

**Comment by AlexDochioiu** [comment]
[Sesori reply] Inspected the review summary and Codex completion notice. Addressed the sole finding by explicitly naming Codex busy-accounting/lifecycle coverage in the notification L2 row.
